lib/dynrender.php

```diff
@@ -691,7 +691,7 @@ function RenderTitleBar($user, $points, $truePoints, $unreadMessageCount, $error
         echo ")";
         echo "</a>";
 
-        if ($permissions >= 3) // 3 == Developer
+        if ($permissions >= 2) // 2 == Jr. Developer

```

Comment:
now would be a good time to change these to use the constants

Reply:
According to Meleu there's issues with using Permissions::JrDeveloper on dynrender. Considering the circumstances I thought it would be acceptable to leave the magic numbers in and change it to how it should be at a later time.

Reply:
ok